src/views/contact-us/contact-us.jsx

```diff
@@ -8,7 +8,7 @@ const render = require('../../lib/render.jsx');
 
 const HelpForm = require('../../components/helpform/helpform.jsx');
 const HelpWidget = require('../../components/helpwidget/helpwidget.jsx');
-const {CONTACT_US_POPUP} = require('../../lib/feature-flags.js');
+const {CONTACT_US_POPUP} = true;
```

Comment:
```suggestion
const CONTACT_US_POPUP = true;
```

It will be `undefined` otherwise because `true` does not have property `CONTACT_US_POPUP`